Diffusion/Common/Views/PromptTextField.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  PromptTextField.swift
+//  Diffusion-macOS
+//
+//  Created by Dolmere on 22/06/2023.
+//  See LICENSE at https://github.com/huggingface/swift-coreml-diffusers/LICENSE
+//
+
+import SwiftUI
+import Combine
+import StableDiffusion
+
+struct PromptTextField: View {
+    @State private var output: String = ""
+    @State private var input: String = ""
+    @State private var typing = false
+    @State private var tokenCount: Int = 0
+    @State var isPositivePrompt: Bool = true
+    @State private var tokenizer: BPETokenizer?
+    @State private var currentModelVersion: String = ""
+
+    @Binding var textBinding: String
+    @Binding var model: String // the model version as it's stored in Settings
+
+    private let maxTokenCount = 75
```

Comment:
```suggestion
    private let maxTokenCount = 77
```

In general this should be `77` I think, see these for reference:
- https://huggingface.co/stabilityai/stable-diffusion-2-1/blob/main/text_encoder/config.json#L16
- https://huggingface.co/runwayml/stable-diffusion-inpainting/blob/main/text_encoder/config.json#L16

Perhaps A1111 uses 75 for start/end tokens (just guessing here), but I don't think that's necessary in our case.

In an ideal world, we'd retrieve the max length from the model's config, but we'll deal with that when we add some class that interfaces with the Hub.

Reply:
Committing this. I did notice the start and end tokens and considered that a loss. 

I am quite looking forward to the HUB interface additions.